x-pack/plugins/rule_registry/server/rule_data_client/rule_data_client.ts

```diff
@@ -215,10 +216,8 @@ export class RuleDataClient implements IRuleDataClient {
           } else {
             this.options.logger.debug(`Writing is disabled, bulk() will not write any data.`);
           }
-          return undefined;
         } catch (error) {
           this.options.logger.error(error);
-
           return undefined;
```

Comment:
Do we need to propagate the error about the failed bulk operation as well?

Reply:
Updated in https://github.com/elastic/kibana/pull/134280/commits/dc17a5a9ce2589d4d216aadbe61c49e6e7df534b